### ISSUE TITLE
Table: Column Resize + Scrollable + Toggle => style not updatet

### ISSUE THREAD

**Issue by MuhSchaf** [primefaces/primeng#11045]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```
https://stackblitz.com/edit/primeng-tablecoltoggle-demo-rp7mhc?file=src/app/app.component.html

**Current behavior**
If you have a _scrollable_ and _resizable_ table, primeng adds a styling for each column in this table twice (One for header- and one for body- columns.
![image](https://user-images.githubusercontent.com/46820453/148739146-fbff7da8-9b4a-48db-a357-25f947386a91.png)

If i toggle now a column - hide it, the style sheet don't change. 
![image](https://user-images.githubusercontent.com/46820453/148746665-72237c1e-066c-438a-9c3f-e848504ace38.png)

There follows, that the columns get the styling of their respective left neighbor:
(Before toggle)
![image](https://user-images.githubusercontent.com/46820453/148748140-e6e4b5d2-4899-4a30-a2cf-6056a5b3235f.png)
(After toggle)
![image](https://user-images.githubusercontent.com/46820453/148748251-7552565d-50bd-4087-9344-0b8041f34bb4.png)

**Expected behavior**
On toggle / updating the _column_-property the style sheet should be updated too. Atleast to the number of columns.

**Minimal reproduction of the problem with instructions**

1. Take the toggle Stackblitz example. 
2. Add the properties of `scrollable="true"` and `resizableColumns="true"` to the p-table, as well as `styleClass="p-datatable-gridlines"`.
3. Add the properties `pResizableColumn` to the th-Tags
4. Resize a column.
5. Toggle a column.

**What is the motivation / use case for changing the behavior?**
My clients hide an show columns while using the application. There are mildly confused on the column-width-change.

**Please tell us about your environment:**

* **Angular version:** 5.X
Version 13.1.2

* **PrimeNG version:** 5.X
12.2.2 (the problem exist still in primeng 13.0.4

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
Chrome/Edge (chromium)
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
5.9

**Comment by yigitfindikli** [comment]
Fixed in 13.1.0.

**Comment by dusto2020** [comment]
This Issue apparently is fixed but in my case it is still happening as OP described it.
